CHANGELOG.md

```diff
@@ -15,7 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-* Add new query parameter "capabilities" in search endpoint [#1054](https://github.com/elastic/package-registry/pull/1054)
+* Add new query parameter "capabilities" in search endpoint [#1054](https://github.com/elastic/package-registry/pull/1054) [#1058](https://github.com/elastic/package-registry/pull/1058)
```

Comment:
Would it be needed to add this PR into the entry related to capabilities ? It has been removed from the `legacyApply` (using `experimental=true`) here in this PR.

Reply:
No need because this is reverting a change that was never released.